getting-started/templates/systemlink-values.yaml

```diff
@@ -673,7 +673,18 @@ fileingestion:
 sl-jupyterhub:
   jupyterhub:
     imagePullSecrets: *pullSecrets
+    hub:
+      extraEnv:
+        ## Setting this to "true" will enable the legacy implementation of Jupyter usernames, which is based on user IDs.
+        JUPYTER_USERNAME_AS_SYSTEMLINK_USER_ID: "false"
     singleuser:
+      cpu:
+        # <ATTENTION> - Use numeric values like .5, 1, 1.5, as Jupyter has this convention for CPU resource allocation.
```

Comment:
Do we expect customers to change these values? We don't call out resource allocations elsewhere in this document. Nothing here tells me why I should change these allocations or what values are reasonable for what workflows. We should either provide the user with that information or else not show these values at this level. This document should be for values that we expect users to need to interact with in common use cases.

Reply:
These resources are for Jupyter user pods, not for services, and it may be expected for the customers to change them. When running notebooks that require lots of RAM and CPU, the pods will break because of this new limitation, which a user already reached inside our company. Specifying them here will help customers change them more easily, especially because of the fact that the resource specification has a different convention that the usual, which should be documented somewhere (as I pointed out in the comments). If we would just tell customers to specify some larger resources, and for example they would just add "3Gi" instead of "3G", all the Jupyter pods will be in a bad state and won't start

Reply:
I agree with JD here. Rather than begin including this information here perhaps we should begin a new values template specific for resources, and include this there.